client/app/account/billing/main/history/billing-main-history.controller.js

```diff
@@ -207,6 +208,20 @@ angular.module('Billing').controller('BillingMainHistoryCtrl', class BillingMain
 
   /* -----  End of EXPORT TO CSV  ------ */
 
+  /* =====================================
+  =            DOWNLOAD BILLS            =
+  ====================================== */
+
+  downloadBills() {
+    this.$uibModal.open({
+      templateUrl: 'account/billing/main/history/downloadBills/billing-main-history-downloadBills.html',
+      controller: 'BillingMainHistoryDonwloadBillsCtrl',
+      controllerAs: '$ctrl',
+    }).result.then(() => {});
```

Comment:
Is returning `{}` really useful ?

Reply:
Yes because of eslint warning "no-unused-expressions"